### PR TITLE
Release 2.9.7-beta1

### PR DESCRIPTION
**Pull request by tidy-dev**
## Description

Looking for the PR for the upcoming first beta of the v2.9.7 series? Well, you've just found it, congratulations!

## Release checklist

- [x] Check to see if there are any errors in Sentry that have only occurred since the last production release
- [x] Verify that all feature flags are flipped appropriately
- Pull request quick view is set to dev
- Notifications beta enabled or macOs
image
- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated
-- Metrics added for notifications

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,20 @@
 {
   "releases": {
+    "2.9.7-beta1": [
+      "[Added] Initial support for high-signal notifications in macOS - #13655",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[New] Initial support for high-signal notifications in macOS - #13655",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+      "[Added] Support pushing workflow files for GitHub Actions to GitHub Enterprise Server - #13640",
+      "[Added] Support CLion as an external editor - #13739. Thanks @Pinzauti!",
+      "[Fixed] Fix close button in full screen mode on macOS - #12838",
+      "[Fixed] Commit message dialog background styles match dialog - #13606",
+      "[Fixed] Job steps on pull-request check run list are no longer intermittently missing - #13531",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Ensure job steps on pull request check run list are always present - #13531",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,20 @@
...
+      "[Fixed] Take alias into account when sorting repositories - #13429",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Take alias into account when sorting repositories - #13429",
```

File: changelog.json [hunk 4]
```diff
@@ -1,5 +1,20 @@
...
+      "[Improved] Support avatars on GitHub Enterprise Server - #13719",
+      "[Improved] Fetch before trying to follow a URL link to a specific branch - #13641. Thanks @Bestra!",
+      "[Improved] Add \"View on GitHub\" context menu option - #13227. Thanks @lhvy!",
+      "[Improved] Signal when a commit summary is getting long - #2055. Thanks @Twixes!",
+      "[Improved] Check run group headers and checks stay in view while scrolling the sub checks or job steps. - #13532",
+      "[Improved] Removed unnecessary punctuation in appearance settings- #13715. Thanks @Pinzauti!"
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Remove unnecessary punctuation in appearance settings- #13715. Thanks @Pinzauti!"
```

**Review by billygriffin — CHANGES_REQUESTED** [review]
Minor changes, but overall looking good!

File: changelog.json [hunk 5]
```diff
@@ -1,5 +1,21 @@
 {
   "releases": {
+    "2.9.7-beta1": [
+      "[New] Initial support for high-signal notifications in macOS - #13655",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[New] Initial support for system notifications when checks fail in macOS - #13655",
```

**Review by billygriffin — DISMISSED** [review]
Wooooo!

File: changelog.json [hunk 6]
```diff
@@ -1,5 +1,21 @@
...
+      "[Added] Support pushing workflow files for GitHub Actions to GitHub Enterprise Server - #13640",
+      "[Added] Support CLion as an external editor - #13739. Thanks @Pinzauti!",
+      "[Fixed] Fix close button in full screen mode on macOS - #12838",
+      "[Fixed] Commit message dialog background styles match dialog - #13606",
+      "[Fixed] Ensure job steps on pull request check run list are always present - #13531",
+      "[Improved] Take alias into account when sorting repositories - #13429",
+      "[Improved] Upgrade to Electron v14.2.3 - #13689",
+      "[Improved] Support avatars on GitHub Enterprise Server - #13719",
+      "[Improved] Fetch before trying to follow a URL link to a specific branch - #13641. Thanks @Bestra!",
+      "[Improved] Add \"View on GitHub\" context menu option - #13227. Thanks @lhvy!",
```
Comment (tidy-dev) [inline changelog.json]:
```suggestion
      "[Improved] Add \"View on GitHub\" context menu option to repository list items - #13227. Thanks @lhvy!",
```

Comment (tidy-dev) [reply changelog.json]: Realized when testing this.. that I had to go to the pr to remember where that was added. :P

**Review by sergiou87 — APPROVED** [review]

**Pull request merged**